src/compiler/facet.ts

```diff
@@ -90,7 +90,7 @@ export function facetMixins(model: Model, marks) {
     // TODO: add properties to make rule optional
     rootMarks.push(getColumnRulesGroup(model, cellWidth));
   } else { // doesn't have column
-    if (model.has(Y)) { //keep y axis in the cell
+    if (model.has(Y)) { // keep y axis in the cell
```

Comment:
It's not but there were only 3 cases where this happened. I'm happy to
remove the rule.

On Fri, Dec 4, 2015 at 2:47 PM Kanit Wongsuphasawat <
notifications@github.com> wrote:

> In src/compiler/facet.ts
> <https://github.com/vega/vega-lite/pull/837#discussion_r46741917>:
>
> > @@ -90,7 +90,7 @@ export function facetMixins(model: Model, marks) {
> >      // TODO: add properties to make rule optional
> >      rootMarks.push(getColumnRulesGroup(model, cellWidth));
> >    } else { // doesn't have column
> > -    if (model.has(Y)) { //keep y axis in the cell
> > +    if (model.has(Y)) { // keep y axis in the cell
>
> I mean it's a good thing but I don't think it is very important.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vega/vega-lite/pull/837/files#r46741917>.
>